fv3core/utils/gt4py_utils.py

```diff
@@ -259,6 +260,71 @@ def make_storage_from_shape(
     return storage
 
 
+storage_shape_outputs = {}
+
+
+@wraps(make_storage_from_shape_uncached)
```

Comment:
why do we need/want this decorator here?

Reply:
To turn the *args **kwargs signature into the real call signature.